filebeat/input/event.go

```diff
@@ -18,9 +18,9 @@ type Event struct {
 	DocumentType string
 	Bytes        int
 	Text         *string
-	JSONFields   common.MapStr
 	JSONConfig   *reader.JSONConfig
 	State        file.State
+	Data         common.MapStr
```

Comment:
why rename to `Data` instead of `Fields`? `Data` sounds quite generic, we need a doc comment saying what data is for?